src/mlp_seq.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, n_dims=1, n_embd=2, n_layer=2, context_len=1, activation=(lam
 
         super(MLPSequence, self).__init__()
 
+        self.name = f"MLP_n={context_len}"
+        self.sequence_model = False
```

Comment:
I think this should be `True`, because it's meant to serve as a sequence model. Where is this property used?

Reply:
I also think this should be true. It's used in eval.py to determine whether to use cuda (attempt to use if True) or cpu (False).